src/libraries/System.Runtime/tests/System/Runtime/DependentHandleTests.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Runtime.Tests
+{
+    // NOTE: DependentHandle is already heavily tested indirectly through ConditionalWeakTable<,>.
+    // This class contains some specific tests for APIs that are only relevant when used directly.
+    public class DependentHandleTests
+    {
+        [Fact]
+        public void GetTarget_ThrowsInvalidOperationException()
+        {
+            Assert.Throws<InvalidOperationException>(() => default(DependentHandle).Target);
+        }
+
+        [Fact]
+        public void SetTarget_ThrowsInvalidOperationException()
+        {
+            Assert.Throws<InvalidOperationException>(() =>
+            {
+                DependentHandle handle = default;
+                handle.Target = new();
+            });
```

Comment:
Nit: any reason this is structured slightly differently from the get test above?  I'd have expected almost the same form:
```suggestion
            Assert.Throws<InvalidOperationException>(() => default(DependentHandle).Target = new());
```

Reply:
That was the original intention, but unfortunately that code doesn't compile 😥

> "error CS0131: The left-hand side of an assignment must be a variable, property or indexer"

See sharplap.io repro [here](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAZgAJcMoBXMDcgERgAcYA7AEw4wAlsuAGxgBYAFABvceRnky5CMABWMeuQAq2KAHMYGabKljZJ8roYBeAHzl2NQYIDcB05T3kJ5AL4vvv3/LEAIxIcigaMFQAFACUvkauMtwAZtj2GFHMbFw8/EIwMQB0mjruFuQoAEzOxrI+Yl5AA==).